Authenticator/Source/TokenListViewModel.swift

```diff
@@ -28,4 +28,11 @@ import Foundation
 struct TokenListViewModel {
     let rowModels: [TokenRowModel]
     let ringProgress: Double?
+    let totalTokens: Int
+    var isFiltering: Bool {
+        return totalTokens > rowModels.count
```

Comment:
At this point the only place `isFiltering` is being used is to determine if the `UITableView` in the `TokenListViewController` should animate row changes. But you are correct that a filter `String` may be set on the `TokenList` but the `TokenListViewModel.isFiltering` will return `false` when the `TokenList.filter` matches all `PersistentToken` instances.

`TokenList` also has an `isFiltering` property that is determined by the value of the `TokenList.filter` property. I can pass that to the `TokenListViewModel` struct on initialization.